### PR TITLE
user settings page issue fixes

### LINKED ISSUE #279: User Settings Page: Extra Border and Spacing Issue Between Sections
On the user settings page, there is an extra border applied. Please remove it, and also ensure there is adequate spacing between the 'Personal Info' and 'Change Password' sections for better clarity and layout.
Screenshot 2024-11-18 at 12 20 19 PM

### PR DESCRIPTION
**Pull request by sadrisya**
A few changes has done in user settings page especially in borders and box shadows ,both mobile and web views have changes
Fixes #279

Screenshot 2024-11-19 at 12 58 24 PM
Screenshot 2024-11-19 at 12 58 38 PM

### CODE REVIEW

**Comment by sadrisya** [comment]
Some more changes is needed as per the new design ,like banner is adding at the  top of the page

File: app/views/settings/_page_body.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div id="settings-page-body-mobile" class="my-6 flex gap-8 rounded border-b p-8 border-line-colour box-shadow-medium">
+<div id="settings-page-body-desktop" class="flex gap-8 border-b p-8 border-line-colour ">
```
Comment (deepakkumarnd) [inline app/views/settings/_page_body.html.erb:1]: change id to  "settings-page-body"

change lines in update.turbo_stream.erb from views/settings and make sure the code works on updating on both mobile and web

File: app/views/settings/_page_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<div id="settings-page-header-desktop" class="flex gap-8 border-b p-8 border-line-colour">
```
Comment (deepakkumarnd) [inline app/views/settings/_page_header.html.erb:1]: rename id to 'settings-page-header' to the same as per the above comment.

File: app/views/settings/_page_body.html.erb (hunk 1 above)
Comment (sadrisya) [reply app/views/settings/_page_body.html.erb]: id changed ,also corresponding changes done in  update.turbo_stream.erb from views/settings

File: app/views/settings/_page_header.html.erb (hunk 2 above)
Comment (sadrisya) [reply app/views/settings/_page_header.html.erb]: id is renamed

**Comment by sadrisya** [comment]
Screenshot 2024-11-25 at 11 31 21 AM

A small change in showing organization ,if the org name is big then showing it like in the above image, previously it was like below

Screenshot 2024-11-25 at 11 33 14 AM

**Pull request merged**